Bio/SearchIO/BlastIO/blast_xml.py

```diff
@@ -189,10 +189,14 @@
 
 
 def _extract_ids_and_descs(raw_id, raw_desc):
-    # Given values of the `Hit_id` and `Hit_def` elements, return a tuple
-    # three elements: all IDs, all descriptions, and the BLAST-generated ID.
-    # The BLAST-generated ID is set to `None` if no BLAST-generated IDs are
-    # present.
+    """Extract IDs, descriptions, and raw BLAST ID from the given raw values.
```

Comment:
Ah, I missed the `PRIVATE` note. It's added now, and I've also shortened the line.